.codeclimate.yml

```diff
@@ -0,0 +1,8 @@
+version: "2"
+exclude_patterns:
+  - "dist/"
+  - "src/api/spec/"
```

Comment:
Have you seen problems with this?

Reply:
No, but this is part of the default exlude_patterns of codeclimate.

Reply:
I see